postorius/Dockerfile.dev

```diff
@@ -18,13 +20,19 @@ RUN set -ex \
 	&& apk add --no-cache --virtual .mailman-rundeps bash sassc \
 		postgresql-client mysql-client py3-mysqlclient curl mailcap \
 		python3 py3-pip libffi gettext \
-	&& python3 -m pip install -U 'Django<3.0' pip setuptools wheel \
-	&& python3 -m pip install git+https://gitlab.com/mailman/postorius@${POSTORIUS_REF} \
+    && python3 -m pip install -U pip setuptools wheel \
+    && python3 -m pip install -U \
+        git+https://gitlab.com/mailman/mailmanclient@${CLIENT_REF} \
+        git+https://gitlab.com/mailman/postorius@${POSTORIUS_REF} \
 		uwsgi \
 		psycopg2 \
 		dj-database-url \
 		mysqlclient \
 		typing \
+		django-utils-six \
+    && python3 -m pip install -U 'Django<3.1' \
```

Comment:
I'd say lets do `Django<3.2`. I am working on bumping that up for rest of the images too. 3.1 has reached EOL already.

Reply:
I've branched from `main` and am trying to keep code consistent after the merge.

I supppse Django upgrade to 3.2 should be done for all the images in the same PR?

Reply:
Do you mean it'd be easy for you to cherry-pick the commits into your fork if all the django version bumps were in the same commit?

Reply:
I'm branching from `main`, not from `bump-django` branch - hence prefer to be consistent with the former. If you've merged `bump-django`  into `main`, then I would target Django 3.2 for consistency.

Reply:
Okay, since this PR is functional enough for me, I'll get this in and we can bump django everywhere in a next PR.